### PR TITLE
weblate: 5.6.2 -> 5.7

### PR DESCRIPTION
**Pull request by erictapen**
## Description of changes

First update to the new Weblate package and module. Builds and deploys on my [test instance](https://weblate.erictapen.name).

Changelogs:

- [weblate: 5.6.2 -> 5.7](https://github.com/WeblateOrg/weblate/releases/tag/weblate-5.7)
- [python3Packages.django-otp: 1.5.0 -> 1.5.1](https://github.com/django-otp/django-otp/blob/master/CHANGES.rst#v151---july-23-2024---admin-search-fields)

New packages:
- python3Packages.cyrtranslit
- python3Packages.django-otp-webauthn

## Things done

- Built on platform(s)
  - [X] x86_64-linux
  - [X] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) (or backporting [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) and [24.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

**Comment by mweinelt** [comment]
> Builds and deploys on my [test instance](https://weblate.erictapen.name).

CSS is 404.

**Comment by erictapen** [comment]
Yeah that's why I switched back into draft mode. I'm investigating.

**Comment by erictapen** [comment]
Now it should work, had to rework the django-compressor settings.

**Comment by erictapen** [comment]
@GrahamcOfBorg test weblate

**Comment by erictapen** [comment]
I noticed that this also unbreaks the current weblate build, as it currently fails due to dependency version bounds.

**Review by Radvendii — APPROVED** [review]
Can't vouch for the weblate details, but the nix side of things LGTM

File: pkgs/by-name/we/weblate/package.nix [hunk 1]
```diff
@@ -47,35 +47,23 @@ python.pkgs.buildPythonApplication rec {
     owner = "WeblateOrg";
     repo = "weblate";
     rev = "weblate-${version}";
-    sha256 = "sha256-t/hnigsKjdWCkUd8acNWhYVFmZ7oGn74+12347MkFgM=";
+    sha256 = "sha256-h5+0lOMD+H0ehtZ0bngA9bI5va1I5KjZH9boaEtXJPo=";
```
Comment (dotlambda) [inline pkgs/by-name/we/weblate/package.nix:50]:
```suggestion
    hash = "sha256-h5+0lOMD+H0ehtZ0bngA9bI5va1I5KjZH9boaEtXJPo=";
```

**Pull request merged**